src/main.c

```diff
@@ -172,6 +174,18 @@ void cin_entry() {
     }
 }
 
+void preload_entry() {
+    Tape t;
+    size_t initTapeSize = 8; // 8 bytes
+    init_tape(&t, initTapeSize);
+    fill_tape(&t, initTapeSize, "test2.txt");
```

Comment:
why is test2.txt hard coded here?

Reply:
forgot to change that, test2.txt should be the argument for the file